Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM metabrainz/python:3.7-20190302 as listenbrainz-base
+FROM metabrainz/python:3.7-20201201 as listenbrainz-base
```

Comment:
as we discussed with CB, ensure that we force install pip 21 in this dockerfile (and others) so that we can test the new version resolver